bin/ch-test

```diff
@@ -1180,6 +1202,16 @@ case $phase in
             dir_perm_rm "$d"
         done
         ;;
+    rootemu)
+        if [[ $scope != full ]]; then
+            if [[ -z "$scope" ]]; then
+                scope="standard"
+            fi
+            printf "error: rootemu incompatible with scope: %s\n" "$scope"
+            exit 1
+        fi
+        test_rootemu "foo"
```

Comment:
Give the argument a meaningful name rather than `foo` (and you don't need quotes here).